server/src/main/java/org/opensearch/index/codec/fuzzy/LongArrayBackedBitSet.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.index.codec.fuzzy;
+
+import org.apache.lucene.store.DataOutput;
+import org.apache.lucene.store.IndexInput;
+import org.apache.lucene.util.Accountable;
+import org.opensearch.common.util.BigArrays;
+import org.opensearch.common.util.LongArray;
+import org.opensearch.common.util.io.IOUtils;
+
+import java.io.Closeable;
+import java.io.IOException;
+
+/**
+ * A bitset backed by a long-indexed array.
+ */
+public class LongArrayBackedBitSet implements Accountable, Closeable {
+
+    private long underlyingArrayLength = 0L;
+    private LongArray longArray;
+
+    LongArrayBackedBitSet(long capacity) {
```

Comment:
The class is marked public but the constructors are marked package-private. Please fix.